codec/text.go

```diff
@@ -0,0 +1,28 @@
+package codec
+
+import (
+        "github.com/shiningrush/droplet/data"
+        "net/http"
+)
+
+type Text struct {
```

Comment:
It would be better to called `Empty`, because this codec should do nothing.

Reply:
fixed